submodules/storage/variables.tf

```diff
@@ -33,8 +33,26 @@ variable "s3_force_destroy_on_deletion" {
   default     = false
 }
 
-variable "s3_encryption_use_sse_kms_key" {
-  description = "if true use 'aws:kms' else 'AES256' for the s3 server-side-encryption."
+variable "ecr_force_destroy_on_deletion" {
+  description = "Toogle to allow recursive deletion of all objects in the ECR repositories. if 'false' terraform will NOT be able to delete non-empty repositories"
   type        = bool
   default     = false
```

Comment:
Should be update our test to make this `true` otherwise we might leak ecr repos

Reply:
I'll create a second PR to set this since the same main.tf.json is used for both sides of the upgrade test